FuzzTesting/Sources/FuzzCommon/fuzz_testing.pb.swift

```diff
@@ -5392,6 +5392,11 @@ extension Fuzz_Testing_Message: SwiftProtobuf.Message, SwiftProtobuf._MessageImp
   }
 }
 
+#if swift(>=5.5)
```

Comment:
You might want to follow the guide from https://github.com/swift-server/guides/blob/main/docs/concurrency-adoption-guidelines.md#if-guarding-code-using-concurrency on these.

I.e. we suggest doing 

```
#if compiler(>=5.5) && canImport(_Concurrency)
```

Reply:
The section [Backwards compatibility of declarations and “checked” Swift Concurrency](https://github.com/swift-server/guides/blob/main/docs/concurrency-adoption-guidelines.md#backwards-compatibility-of-declarations-and-checked-swift-concurrency) says to use `swift(>=5.5)` instead. I'm happy to drop in the `canImport`, however.

Reply:
Just out of curiosity, what's the failure case for not having `canImport`, since `Sendable` lives in the stdlib?

Reply:
The question is also made relevant and not a curiosity by @thomasvl's ask to group up conformances in the generated code.

Reply:
That trick is necessary to survive situations in which one would build this project with a NEW compiler (5.5+) but targeting an SDK that does not have concurrency.

Reply:
> That trick is necessary to survive situations in which one would build this project with a NEW compiler (5.5+) but targeting an SDK (mac, ios etc) that does not have concurrency.

Perfect, thanks!

Reply:
> I can definitely improve the wording if you found it confusing though

The document wasn’t confusing but it contradicts your suggestion. Your initial comment said to use `compiler(>=5.5.)` (“i.e., we suggest doing”) and linked to the document. The document says to use `compiler(>=5.5)` for concurrency declarations in code, then later says `swift(>=5.5)` for adding `Sendable` conformances. After updates, this PR now follows what the document says: `#if swift(>=5.5) && canImport(_Concurrency)`.